tox.ini

```diff
@@ -19,7 +21,9 @@ deps =
     Flask-SQLAlchemy
     future
     # the sdk doesn't support earlier version of django
-    django >= 1.10, <2.0
+    py{27,34}-!django2-!django3,django1: django >= 1.10, <2.0
```

Comment:
You won't probably need to specify the supported django version for each runtime. If you just do
`py{27,34,35,36,37,38}: django >= 1.10`, tox should automatically pick up the highest version for each python runtime. Please correct me if I'm wrong.

Reply:
You're right, tox is smart enough to rely on pypi classifiers.

Thanks for the suggestion.